sdk/src/mcm_dp.c

```diff
@@ -267,6 +267,15 @@ mcm_conn_context* mcm_create_connection(mcm_conn_param* param)
         }
         conn_ctx->session_id = 0;
         break;
+    case PROTO_TCP:
+        log_warn("Unsupported protocol: TCP (%d)", param->protocol);
+        break;
+    case PROTO_HTTP:
+        log_warn("Unsupported protocol: HTTP (%d)", param->protocol);
+        break;
+    case PROTO_GRPC:
+        log_warn("Unsupported protocol: GRPC (%d)", param->protocol);
+        break;
```

Comment:
Could you elaborate more on why we need these cases instead of the existing default printing?

Reply:
Existing default printing is theoretically enough, but not very descriptive, e.g. `WARN - Unsupported protocol: 3` does not say much about which protocol is supposedly unsupported. I know it may sound like one step too far, but it may shorten the debugging time.

Reply:
Mmm, it's redundant, really.

Reply:
I'll be honest, I was quite confused when I encountered that for the first time. Is there any standard that numbers these protocols in the same manner?

If we really want to get back to the previous solution, what if we skip the number, leaving `WARN - Unsupported protocol`, or at least put it in the bracket `WARN - Unsupported protocol (3)`?

Reply:
I vote for `WARN - Unsupported protocol (3)`

Reply:
> I vote for `WARN - Unsupported protocol (3)`

Done in 61e3366.